packages/plantae/src/ky/createKyHooks.spec.ts

```diff
@@ -0,0 +1,230 @@
+import ky from "ky";
+import { describe, expect, test } from "vitest";
+
+import {
+  abortSignalPlugin,
+  afterResponsePlugin,
+  beforeRequestPlugin,
+  firstPlugin,
+  headerSetPlugin,
+  headerSetRequestReseponsePlugin,
+  modifiedHeaderResponsePlugin,
+  modifiedResponseBodyPlugin,
+  modifyUrlPlugin,
+  postMethodPlugin,
+  postMethodWithBodyPlugin,
+  secondPlugin,
+} from "../__mock__/plugin";
+import createKyHooks from "./createKyHooks";
+
+describe("ky:beforeRequest -", () => {
+  test("headers", async () => {
+    const hooks = createKyHooks({
+      client: ky,
+      plugins: [headerSetPlugin()],
+    });
+
+    const kyWithHoks = ky.extend({
```

Comment:
```suggestion
    const kyWithHooks = ky.extend({
```

Please fix trivial typo issues.